W-18564672 a2a pii detector policy mb (#863)
mulesoft/docs-gateway: gateway/1.9/modules/ROOT/pages/policies-included-a2a-pii-detector.adoc

== Configuring Policy Parameters

include::partial$policy-title-headers.adoc[tag=configFileTitleOnly]

> The A2A PII Detector policy is not supported in Local Mode.

Author: ```suggestion
The A2A PII Detector policy isn't supported in Local Mode.
```
